docs/content/app-protect/configuration.md

```diff
@@ -203,3 +203,171 @@ spec:
   softwareVersion: 15.1.0
   tag: Fruits
 ```
+
+## OpenAPI Specification in NGINX Ingress Controller
+
+The OpenAPI Specification defines the spec file format needed to describe RESTful APIs. The spec file can be written either in JSON or YAML. Using a spec file simplifies the work of implementing API protection. Refer to the [OpenAPI Specification](#https://github.com/OAI/OpenAPI-Specification) (formerly called Swagger) for details. 
+
+NGINX Ingress Controller supports OpenAPI Specification versions 2.0 and 3.0.
+
+The simplest way to create an API protection policy is using an OpenAPI Specification file to import the details of the APIs. If you use an OpenAPI Specification file, NGINX App Protect WAF will automatically create a policy for the following properties (depending on what's included in the spec file):
+* Methods
+* URLs
+* Parameters
+* JSON profiles 
+
+An OpenAPI-ready policy template is provided with the NGINX App Protect WAF packages and is located in: `/etc/app_protect/conf/NginxApiSecurityPolicy.json`
+
+It contains violations related to OpenAPI set to blocking (enforced).
+
+### Types of OpenAPI References
+
+There are different ways of referencing OpenAPI Specification files. The configuration is similar to [External References](#external-references).
+
+**Note**: Any update of an OpenAPI Specification file referenced in the policy will not trigger a policy compilation. This action needs to be done actively by reloading the NGINX configuration.
+
+#### URL Reference
+
+URL reference is the method of referencing an external source by providing its full URL.
+
+Make sure to configure certificates prior to using the HTTPS protocol - see the [External References](#https-reference) for details.
+
+## Configuration in NGINX Ingress Controller
+
```

Comment:
Add a line: "These are the typical steps to deploy an OpenAPI protection Policy in NGINX Ingress Controller:"

Reply:
done